src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
> Circular means that you can't walk off the end, that you can follow the next (or prev) pointers forever rather than eventually hitting null.

Oops. Yeah, doubly-linked means there's a forward and back link instead of just a forward link. Temporary brain malfunction :smile: